catalyst/contrib/utils/misc.py

```diff
@@ -56,4 +56,18 @@ def args_are_not_none(*args: Optional[Any]) -> bool:
     return True
 
 
-__all__ = ["args_are_not_none", "make_tuple", "pairwise"]
+def find_all(x: Iterable[Any], x0: Any) -> List[int]:
```

Comment:
is it possible to find more easy-to-go name? this one looks too general

Reply:
done